### ISSUE TITLE
Silent Install of 2022.3.1 Unsuccessful 

### ISSUE THREAD

**Issue by snowflake-tvi** [nvaccess/nvda#14306]
Our network admin is trying to do a silent install of NVDA 2022.3.1 and has encountered issues.  He says when the push install goes out the startup music plays but NVDA doesn't install.  He also says he's using the command line --install-silent --enable-start-on-logon=True  but the install still wants UAC intervention.  We'd like to avoid this if we can since it requires a complex password for each computer individually, and we have about 100 PCs on our campus.  NVDA 2022.3 installs fine, but a popup about an available update comes up and we'd like to avoid this issue as it can confuse students (i.e. do I update, it tells me to, why can't I update, will NVDA still work, etc).  Can anyone tell me if there is a way around this issue?
Thanks!
Am

**Comment by feerrenrut** [comment]
It would be best if the network admin could contact us directly via info@nvaccess.org I expect there should be a way to pre-authorize the UAC intervention when doing a push install. This certainly doesn't sound like a problem unique to NVDA.

As it stands this issue isn't actionable for us, I'm going to close it.
These kinds of questions are better to be directed to the [projects "Discussions"](https://github.com/nvaccess/nvda/discussions), our [mailing lists (see project readme)](https://github.com/nvaccess/nvda#communication-channels), or [directly with NV Access](mailto:info@nvaccess.org).
